api/interservice/authz/v2/project.proto

```diff
@@ -32,7 +32,9 @@ message GetProjectResp {
     Project project = 1;
 }
 
-message ListProjectsReq {}
+message ListProjectsReq {
+    bool globalFilterView = 1;
```

Comment:
⚠️ let's investigate why `protoc-gen-lint` doesn't yell at you there -- it should be `global_filter_view`, and it should be raised [like this one](https://buildkite.com/chef-oss/chef-automate-master-verify/builds/1677#fd0dc244-d0c7-4d0c-b0ad-750a34bcec7e/326-708)